Patch-Source: https://github.com/ryanoasis/nerd-fonts/pull/820
--
From 68130bd498709d782e9a51f34bbb4df7a7c3ddd2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tor=20Arne=20Vestb=C3=B8?= <torarnv@gmail.com>
Date: Sun, 10 Apr 2022 15:33:11 +0200
Subject: [PATCH] font-patcher: Ensure PostScript font name does not contain
 spaces

When pulling the subfamily out of the sfnt_names SubFamily property,
we will get a subfamily with possible spaces, e.g. 'Bold Italic'.

When constructing the final unique font name (PostScript name), we
need to remove those spaces, to make the font name valid, otherwise
the font will fail validation with a warning when installing.

Fixes #413
---
 font-patcher | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/font-patcher b/font-patcher
index 7d063ab24..fe3122b5c 100755
--- a/font-patcher
+++ b/font-patcher
@@ -343,8 +343,9 @@ class font_patcher:
                 familyname += " Mono"
 
         # Don't truncate the subfamily to keep fontname unique.  MacOS treats fonts with
-        # the same name as the same font, even if subFamily is different.
-        fontname += '-' + subFamily
+        # the same name as the same font, even if subFamily is different. Make sure to
+        # keep the resulting fontname (PostScript name) valid by removing spaces.
+        fontname += '-' + subFamily.replace(' ', '')
 
         # rename font
         #
